components/clinical/plan-definition-editor/src/index.tsx

```diff
@@ -0,0 +1,240 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import { PlanDefinition } from '@ltht-react/types'
+import { BANNER_COLOURS, WIDESCREEN_MINIMUM_MEDIA_QUERY } from '@ltht-react/styles'
+import { BullseyeIcon } from '@ltht-react/icon'
+import { Toggle as ToggleInput } from '@ltht-react/input'
+import { ChangeEvent, FC, useMemo } from 'react'
+
+const toggleRowWidth = '3.5rem'
+
+const toggleInputMixin = css`
+  > div:first-of-type {
+    text-align: right;
+    width: ${toggleRowWidth};
+    padding-right: 1rem;
+  }
+`
+
+const StyledProblemToggleRow = styled.div`
+  display: flex;
+  font-weight: bold;
+  align-items: center;
+  ${toggleInputMixin}
+  margin-bottom: 0.25rem;
+`
+
+const StyledEducationToggleRow = styled.div`
+  display: flex;
+  align-items: center;
+  ${toggleInputMixin}
+  margin-bottom: 0.25rem;
+
+  > label svg {
+    margin-right: 0.5rem;
+  }
+`
+
+const StyledTargetRow = styled.div`
+  display: flex;
+  align-items: center;
+  ${toggleInputMixin}
+
+  svg {
+    align: right;
+    margin-right: 1rem;
+  }
+
+  > span:first-of-type {
+    text-align: right;
+    display: block;
+    width: ${toggleRowWidth};
+  }
+
+  > div:first-of-type {
+    flex: 1;
+    text-align: left;
+  }
+`
+
+const StyledListItem = styled.li`
+  margin-bottom: 1rem;
+
+  ${WIDESCREEN_MINIMUM_MEDIA_QUERY} {
+  }
+`
+
+const StyledSectionToggle = styled.div`
+  background-color: ${BANNER_COLOURS.INFO.BACKGROUND};
+  font-weight: bold;
+  color: black;
+  display: flex;
+  align-items: center;
+  padding: 0.5rem 0.25rem;
+  ${toggleInputMixin}
+`
+
+const StyledBullseyeIcon = styled(BullseyeIcon)``
+
+const StyledProblemSection = styled.ul`
+  list-style: none;
+  padding: 0.25rem 0.25rem;
+  margin-bottom: 1rem;
+`
+
+export const SelectedPlanDetail: FC<ISelectedPlanDetailProps> = ({
+  planDefinition,
+  selectedProblemIds,
+  onProblemChange,
+}) => {
+  // Split actions from plan definition into clinical problem based and education based
+  const { problemActions, educationActions } = useMemo(() => {
+    const pa = planDefinition.action?.filter(
+      (action) =>
+        action?.code && action?.code[0] && action?.code[0].coding && action?.code[0].coding[0]?.code === 'problem'
+    )
+    const ea = planDefinition.action?.filter(

```

Comment:
Very minor one but I have a slight preference for renaming these to `educationActions` and then assigning them as shorthand properties in the return type. Just for readability!